cmd/mdatagen/metrics_v2.tmpl

```diff
@@ -162,6 +163,7 @@ func With{{ $name.Render }}(val {{ $attr.Type.Primitive }}) ResourceOption {
 // just `Emit` function can be called instead. Resource attributes should be provided as ResourceOption arguments.
 func (mb *MetricsBuilder) EmitForResource(ro ...ResourceOption) {
 	rm := pdata.NewResourceMetrics()
+	rm.SetSchemaUrl(conventions.SchemaURL)
```

Comment:
Is this the only line where `conventions` package is used? Presumably the same imported conventions package should be use somewhere else in the code to refer to some actual semantic conventions and add some Resource or Metrics attributes and that code should use the same imported package so that the SchemaURL and the attribute names match each other.
Where in the code is that usage?

Reply:
As I replied below, `conventions` package will be used in the following PR https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/8840